### DISCUSSION TITLE
Tipo de datos en planes académicos (E1)

### DISCUSSION THREAD

**Discussion by dgolzm** [IIC2413/Syllabus-2024-2#124] [General]
Hola, queria consultar de qué tipo de datos tomamos para diferenciar entre cursos mínimos, ofgs y oprs, ya que en el plan de estudio no queda claro si se guardan como un array de cursos con sus datos especificos, o simplemente como un int indicando la cantidad de cursos de ese tipo.

**Comment by VicenteSanMartin** [comment]
Hola! Te pido que realices tus consultas mediante [issues](https://github.com/IIC2413/Syllabus-2024-2/issues). Las dicussions serán abiertas por el cuerpo docente para hacer aclaraciones o resolver dudas frecuentes.
